lib/pbench/server/database/__init__.py

```diff
@@ -0,0 +1,16 @@
+from pbench.server.database.models import *  # noqa
```

Comment:
Pretty sure this won't do what you want. (Unless all of the model objects were in the `__init__.py`, which some people do recommend.)

But take a look at https://docs.pylonsproject.org/projects/pyramid_cookbook/en/latest/database/sqlalchemy.html#importing-all-sqlalchemy-models for some ideas to automatically import from the module.

Reply:
Its interesting, the pyramid package installation recommendation is a little weird though, they seem to suggest installing it like `$VENV/bin/pip install "pyramid==2.0"` instead of bin/activate, then pip3. This would however adds an additional dependency, for now I will just add the model imports in the model's `__init__` file and with a note that every new model defined we need to add an import statement here

Reply:
I guess we can use it by keeping the settings parameter default 
```
config = Configurator()
config.scan('myapp.models') 
```

Reply:
If possible, you should specify which diagnostic we want to suppress with the `# noqa` directive.  (I think Dave said it was `F401`.)

Reply:
Yeah I thought about doing that too, and now that I think about it again I feel in the process of making it straight forward I took a long route to the destination